### ISSUE TITLE
x10.conf-LAT-LONG ENTRIES NOT RECOGNISED for Dawn dusk calculation

### ISSUE THREAD

**Issue by npaisnel** [HeyuX10Automation/heyu#15]
Just been looking at my x10conf file and other commands for heyu

In my conf I have
`TTY     /dev/ttyUSB0 `              
`START_ENGINE AUTO`
`HOUSECODE         A`
` DATE_FORMAT DMY`
` LAUNCH_SOURCE ANYSRC`
`LONGITUDE W002:3.14
` LATITUDE N49:10.12`

I have tried various formats of the lat and long, and then do  _**heyu upload**_.  Upload is a success
Yet when I do 
_**heyu show da**_
 I get 

`pi@x10:/etc/heyu $ heyu show da`
`LONGITUDE and LATITUDE must be specified in config file.`
`pi@x10:/etc/heyu $`

It is specified yet does not seem to be seen or used

**Comment by npaisnel** [comment]
ignore me .
I am an idiot
I had not done
`heyu restart'

**Comment by heyu-users** [comment]
On Thursday 16 August 2018 07:58:48 npaisnel heyu.github@yahoo.com 
[heyu_users] wrote:

> Just been looking at my x10conf file and other commands for heyu
>
> In my conf I have
> `TTY     /dev/ttyUSB0
> START_ENGINE AUTO
> HOUSECODE         A
> DATE_FORMAT DMY
> LAUNCH_SOURCE ANYSRC
> LONGITUDE W002:3
> LATITUDE N49:10
> `
>
> **_I have tried various formats of the lat and long, and then do_**
> heyu upload.  Upload is a success Yet when I do
> _**heyu show da**_
>  I get
>
> `pi@x10:/etc/heyu $ heyu show da
> LONGITUDE and LATITUDE must be specified in config file.
> pi@x10:/etc/heyu $`
>
> It is specified yet does not seem to be seen or used

from my /etc/heyu/x10-conf:
# Heyu version 2 uses prefixes N|S or E|W for Latitude and Longitude.
LONGITUDE       W80:52
LATITUDE        N39:41

And it works:
gene@coyote:/etc/heyu$ heyu show da
Dawn = 06:37 EDT  Dusk = 20:18 EDT  Sunrise/Sunset

I think it may be that it expects the first digit is non-zero, but do not 
know for sure. OTOH, this is a cm11a. And I've never seen a cm12 which 
is a different critter entirely.

-- 
Cheers, Gene Heskett
--
"There are four boxes to be used in defense of liberty:
 soap, ballot, jury, and ammo. Please use in that order."
-Ed Howdershelt (Author)
Genes Web page <http://geneslinuxbox.net:6309/gene>

**Comment by npaisnel** [comment]
I have confirmed that this:

LONGITUDE W002:3
 LATITUDE N49:10
does work.

it was my error..  either lacking a eyu restart or heyu engine not running.
I have since set heyu engine to start as service. since setting

`START_ENGINE AUTO`

does not seem to start the engine.

CM12 is basically the same as the CM11, just that it has the British plug and socket and 240 volt capable, other wise I am lead to believe it is identical, unlike the CM15 which **is** a different beast.
